xbmc/dbwrappers/qry_dat.cpp

```diff
@@ -890,55 +778,55 @@ void field_value::set_asInt64(const int64_t i)
   field_type = ft_Int64;
 }
 
-fType field_value::get_field_type()
+fType field_value::get_field_type() const
 {
   return field_type;
 }
 
-std::string field_value::gft()
+std::string field_value::gft() const
```

Comment:
function gft() doesn't seem to have callers. Might as well remove it.

Reply:
At first I was going to comment on the weird structure of the function - then noticed it's not even used.
I'd say it's unlikely to be used, except maybe for logging. If it's to stay for now, then should use the more compact and usual
```
case ft_String:
  return "string";
case ft_Boolean
  return "bool";
etc...
```

or did I miss something?
it's not like a std::string can "adopt" an existing C-style string with assign, it still makes a copy.
trying to guess, maybe the old Kodi custom string type had that feature, but no more
This function could return a string_view actually (no callers, so doesn't break anything :-) )